src/tools/ceph_objectstore_tool.cc

```diff
@@ -1069,7 +1070,23 @@ int dump_data(Formatter *formatter, bufferlist &bl)
   formatter->open_object_section("data_block");
   formatter->dump_unsigned("offset", ds.offset);
   formatter->dump_unsigned("len", ds.len);
-  // XXX: Add option to dump data like od -cx ?
+  if (!dump_data_path.empty()) {
+    int fd = open(dump_data_path.c_str(), O_WRONLY|O_CREAT|O_LARGEFILE, 0666);
+    if (fd == -1) {
+      std::cerr << "open " << dump_data_path << " failed: "
+                << cpp_strerror(errno) << std::endl;
+    } else {
+      int ret = pwrite(fd, ds.databl.c_str(), ds.len, ds.offset);
```

Comment:
Hmm, I'm not 100% sure but I think I can recall some methods in `bufferlist` for dumping its content to a file. Perhaps we could reuse?

Reply:
@rzarzynski Thanks. Updated